README.md

```diff
@@ -197,11 +199,14 @@ See [Bamboo.EmailPreviewPlug](https://hexdocs.pm/bamboo/Bamboo.EmailPreviewPlug.
 
 ## Mandrill Specific Functionality (tags, merge vars, etc.)
 
-Mandrill offers extra features on top of regular SMTP email like tagging, merge vars, and scheduling emails to send in the future. See [Bamboo.MandrillHelper](https://hexdocs.pm/bamboo/Bamboo.MandrillHelper.html).
+Mandrill offers extra features on top of regular SMTP email like tagging, merge
+vars, and scheduling emails to send in the future. See
+[Bamboo.MandrillHelper](https://hexdocs.pm/bamboo/Bamboo.MandrillHelper.html).
 
 ## Testing
 
-You can use the Bamboo.TestAdapter along with [Bamboo.Test] to make testing your emails a piece of cake.
+You can use the Bamboo.TestAdapter along with [Bamboo.Test] to make testing your
+emails a piece of cake.
```

Comment:
I like the general idea. I’m not a fan of easy though because it’s one of those words user so much that I feel it’s lost it’s meaning. Maybe there is something else that can be used?

* make testing straightforward
* make testing less error prone
* make testing less of a headache

?

> On Apr 1, 2016, at 2:03 PM, Jason Draper <notifications@github.com> wrote:
> 
> In README.md <https://github.com/thoughtbot/bamboo/pull/108#discussion_r58243636>:
> 
> >  
> >  ## Testing
> >  
> > -You can use the Bamboo.TestAdapter along with [Bamboo.Test] to make testing your emails a piece of cake.
> > +You can use the Bamboo.TestAdapter along with [Bamboo.Test] to make testing your
> > +emails a piece of cake.
> Minor: what do you think about using easy here instead of a piece of cake. I don't think the reference makes any more distinction and I could imagine a lot of non-english natives not understanding that temr.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/thoughtbot/bamboo/pull/108/files/6b83f69c55d86ac47f88f7bb619198ea3e1e89fa#r58243636>